config/routes.rb

```diff
@@ -29,5 +30,7 @@
   resources :forgot_passwords, only: [:create]
   get 'forgot_password_confirmation', to: 'forgot_passwords#show'
   resources :password_resets, only: [:show, :create]
-  get 'expired_token', to: 'password_resets#expired_token'
+  get 'expired_token', to: 'pages#expired_token'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.